### ISSUE TITLE
Feature Request: Namespace framework assets - frameworks often contain assets with the same file name

### ISSUE THREAD

**Issue by sciascia** [bdkjones/CodeKit-1-Old#135]
Both Compass and Compass Blueprint contain a _utilities.scss file - so adding them to Codekit seems to cause errors.

Also, a lot of frameworks seem to include duplicate asset files within the same framework - normally within an "examples" directory but also via a sass file that might import another file of the same name from deeper within the framework.

e.g. 
I'm trying to add this directory: /compass/frameworks/compass/stylesheets/compass/ 
From the Compass Git repo here: https://github.com/chriseppstein/compass

But get duplication errors.

It would be very useful if Codekit namespaced asset files so they became unique and maybe let you choose which files to use?

Or if Codekit allowed you to change the framework name, you could add directories from deeper within the framework and rename them so they make more sense.

e.g. 
Add /compass/frameworks/compass/stylesheets/compass/utilities but rename the framework within Codekit to "compass-utilities" so you can see which framework the directory belongs to.

Awesome app BTW - friggin' nice work!

**Comment by bdkjones** [comment]
First-class Compass support is coming. You won't have to add Compass files to a framework in order to use Compass.

The existing frameworks feature is really designed for developers to build their own frameworks.

**Comment by sciascia** [comment]
No worries matey.

That framework feature would fixe one thing I haven't figured out yet - how to incorporate frameworks (like Twiiter Bootstrap or Zurb Foundation) without modifying the original or resorting to overrides and !important declarations. And being able to pull the latest version down from Github and have it update across all projects is the other thing it would resolve beautifully.

Basically, a godsend to us designers who use SASS/Compass but don't use Ruby. We are just designers after all!

Cheers
